R/Simulations-methods.R

```diff
@@ -528,6 +528,12 @@ setMethod("summary",
       propAtTarget <- mean((toxAtDoses > target[1]) &
         (toxAtDoses < target[2]))
 
+      if (.hasSlot(object, "stop_report") == TRUE) {
```

Comment:
I think that every object of class `Simulations` should have this slot. Values between the instances may differ but every instance should have this slot, I guess.